docs/docs/Components/components-agents.md

```diff
@@ -45,37 +48,99 @@ For more information on this component, see the [tool calling agent documentatio
 | handle_parsing_errors | Boolean | Whether to handle parsing errors during agent execution. |
 | verbose | Boolean | Enables verbose output for detailed logging. |
 
-### Outputs
+**Outputs**
 
 | Name | Type | Description |
 |------|------|-------------|
 | response | Message | The agent's response to the given input task. |
 
-## CSV Agent
+</details>
 
-This component creates a CSV agent from a CSV file and LLM.
+## Legacy components
+
+Legacy components are available to use but no longer supported.
+
+### JSON Agent
+
+This component creates a JSON agent from a JSON or YAML file and an LLM.
 
-### Inputs
+<details>
+<summary>Parameters</summary>
+
+**Inputs**
 
 | Name | Type | Description |
 |------|------|-------------|
 | llm | LanguageModel | Language model to use for the agent |
-| path | File | Path to the CSV file |
-| agent_type | String | Type of agent to create (zero-shot-react-description, openai-functions, or openai-tools) |
+| path | File | Path to the JSON or YAML file |
```

Comment:
Add periods and articles to descriptions. Apply this throughout. I won't comment on the rest of the instances where this fix needs to be made.